bigframes/dataframe.py

```diff
@@ -3106,6 +3107,12 @@ def to_gbq(
                 + constants.DEFAULT_EXPIRATION,
             )
 
+        if len(labels) != 0:
```

Comment:
We probably want to add a `system` test so that the test coverage didn't get down:

https://screenshot.googleplex.com/73WshjpaD6NZpjD from https://fusion2.corp.google.com/ci/kokoro/prod:bigframes%2Fpresubmit%2Fpresubmit/activity/90e96fb7-f11b-4590-aab4-e1563516b6b2/log

Reply:
done!